### DISCUSSION TITLE
add a toggle for the routing animation

### DISCUSSION THREAD

**Discussion by slonkazoid** [bitwarden/clients#9995] [Password Manager]
### Code Contribution Proposal

there is currently a really annoying 400ms animation whenever you click on anything in the extension. this is extremely annoying and an accessibility issue  
there is already an [old and exceedingly popular community thread](https://community.bitwarden.com/t/option-to-adjust-or-disable-ui-animation/10614) about this  
i have [already written the code](https://github.com/slonkazoid/bitwarden-clients/commit/57fe1239dcb3510938914aa32a55ef026bab85d2), but keep in mind i have never written any bitwarden or Angular code ever  
i would like to contribute this change in a pull request

**Comment by slonkazoid** [comment]
I have made a pull request for this: https://github.com/bitwarden/clients/pull/10005

**Comment by djsmith85** [comment]
Marking this as resolved. Thanks again @slonkazoid for you contribution!
